app/models/suggestion.py

```diff
@@ -0,0 +1,120 @@
+from .. import db
+from datetime import datetime
+import pytz
+
+
+class Suggestion(db.Model):
+    """
+    Association between a resource and potential suggestions for it
+    """
+    __tablename__ = 'suggestions'
+    id = db.Column(db.Integer, primary_key=True)
+    # suggestion_type is 0 for insertion, 1 for edit, and 2 for deletion
+    suggestion_type = db.Column(db.Integer)
```

Comment:
remove suggestion_type; to distinguish suggestions for new resources, we'll make resource_id of these suggestion -1 (and please add comment for that)